README.md

```diff
@@ -127,7 +127,7 @@ Options:
   --debug            log debug information to stderr.
   -v, --version      show version information 
   -h, --help         show help 
-
+  --changeset        helpful switch to verify changes
```

Comment:
This is only available on certain commands. The `--help` output for those commands will show it.